src/stateManager/StateManager.ts

```diff
@@ -566,29 +588,70 @@ class StateManager {
         return this.getEncodedState().then(ethers.keccak256);
     }
 
-    private handleJoinChannelStorage(joinChannelBlock: JoinChannelBlockStruct) {
-        const blockHash = ethers.keccak256(
-            Codec.encode(joinChannelBlock, Type.JoinChannelBlock)
+    private async calculateTotalBalance(
+        channels: { balance: BalanceStruct }[],
```

Comment:
would call this still `balances` since you're only interested in that property - was confusing on first read